jaeger-production-template.yml

```diff
@@ -36,7 +36,6 @@ items:
         - image: jaegertracing/jaeger-collector:1.2
```

Comment:
This image is still *not* following the changes from jaegertracing/jaeger#815. You'll need a new version of those images and then change the YAML files here to use this new version.